content/en/platform/corda/5.0/introduction/about-the-docs.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "About the Docs"
+date: 2023-04-21
+menu:
+  corda5:
+    identifier: corda5-about-the-docs
+    parent: corda5-intro
+    weight: 1000
+section_menu: corda5
+---
+
+The following topics are explained on this page:
+* [Documenting Product Editions](#documenting-product-editions)
+* [End of Life Strategy](#end-of-life-strategy)
+* [Reporting an Error in the Docs](#reporting-an-error-in-the-docs)
+
+## Documenting Product Editions
+The documentation relating to Corda 5 is structured differently than in previous versions of Corda. Previously, Corda Enterprise and Corda Community content was published in separate documentation sets. 
+From Corda 5.0 onwards, there is one documentation set, but the features that relate to Corda Enterprise are marked to avoid confusion. 
+
+{{< 
+  figure
+	 src="enterprise-icon.png"
+   width=55%
+	 figcaption="Documentation heading with Enterprise icon"
```

Comment:
see my next comment about the filename `enterprise-icon.png`

Reply:
Changed this one.